_artifacts/document-attributes.adoc

```diff
@@ -1,8 +1,9 @@
 
-:REBUILT: Thursday, November 05, 2020
+:REBUILT: Tuesday, December 22, 2020
 
-:ENTERPRISE_VERSION: 7.10
-:ENTERPRISE_VERSION_LONG: 7.10.0
+
+:ENTERPRISE_VERSION: 7.9
+:ENTERPRISE_VERSION_LONG: 7.9.1
```

Comment:
Why would you do this for master? Or is this strictly temporary, to remove confusion in SME reviews?

Reply:
Yes, I will manually fix this when I merge with the repo. I commented on this in the ba chat today.